### ISSUE TITLE
Multiple files with the same name results in only one file being imported

### ISSUE THREAD

**Issue by calmyournerves** [glromeo/esbuild-sass-plugin#18]
Hey there!

We noticed a problem when multiple `.scss` files with the same name but in different folders are imported via JS. The resulting output file only includes the styles of one of the files.

#### **`entrypoint.js`**
``` js
import './component_a'
import './component_b'
```

#### **`component_a/index.js`**
``` js
import './style.scss'
```

#### **`component_a/style.scss`**
``` css
.component_a {
  background: blue;
}
```

#### **`component_b/index.js`**
``` js
import './style.scss'
```

#### **`component_b/style.scss`**
``` css
.component_b {
  background: yellow;
}
```

### Expected Output

```css
/* sass:component_a/style.scss */
.component_a {
  background: blue;
}

/* sass:component_b/style.scss */
.component_b {
  background: yellow;
}
```

### Actual Output

```css
/* sass:./style.scss */
.component_b {
  background: yellow;
}
```

### Output from esbuild without plugin

When running it without `esbuild-sass-plugin` and regular CSS, `esbuild` seems to do it correctly and outputs the following:

```css
/* component_a/style.css */
.component_a {
  background: blue;
}

/* component_b/style.css */
.component_b {
  background: yellow;
}
```

## Possible Solution

I played around with the compiled source and was able to get it working correctly with the following change in `plugin.js` inside `build.onResolve` which returns the path relative to the `basedir`:

```js
build.onResolve({ filter: /\.(s[ac]ss|css)$/ }, useExclude((args) => {
    const fullPath = path_1.resolve(args.resolveDir, args.path)
    const relativePath = fullPath.replace(`${options.basedir}/`, '')
    return { path: relativePath, namespace: "sass", pluginData: args };
}));
```

After this change, the output is as follows:

```css
/* sass:component_a/style.scss */
.component_a {
  background: blue;
}

/* sass:component_b/style.scss */
.component_b {
  background: yellow;
}
```

Let me know if I missed something or can help any further. If needed, I can also create a reproduction repository.

Cheers!

**Comment by calmyournerves** [comment]
Oh and I forgot to add: Thanks for creating this plugin, it helps us a lot 🙂

**Comment by glromeo** [comment]
Hi @calmyournerves thank you for your help in getting to the bottom of this issue, I do really appreciate it! 

I have a fix for it in [v1.4.8](https://www.npmjs.com/package/esbuild-sass-plugin)

**Comment by calmyournerves** [comment]
Hey @glromeo

Thanks so much for fixing this so quickly!
